src/API/Helpers/InformationHeaders.php

```diff
@@ -38,62 +50,63 @@ public function setEnvProperty($name, $version)
     }
 
     /**
-     * TODO: Deprecate, not used.
+     * TODO: Deprecate and remove from all dependant libs.
+     *
+     * @param string $name    Dependency or platform name.
+     * @param string $version Dependency or platform version.
      *
-     * @param string $name
-     * @param string $version
+     * @return void
      *
      * @codeCoverageIgnore - Slated for deprecation
      */
     public function setEnvironment($name, $version)
     {
-        $this->data['environment'][] = [
-            'name' => $name,
-            'version' => $version,
-        ];
+        $this->setEnvProperty($name, $version);
     }
 
     /**
-     * TODO: Deprecate, not used.
+     * Replace the current env data with new data.
      *
-     * @param array $data
+     * @param array $data Env data to add.
      *
-     * @codeCoverageIgnore - Slated for deprecation
+     * @return void
      */
-    public function setEnvironmentData($data)
+    public function setEnvironmentData(array $data)
     {
-        $this->data['environment'] = $data;
+        $this->data['env'] = $data;
     }
 
     /**
      * TODO: Deprecate, not used.
      *
-     * @param string $name
-     * @param string $version
+     * @param string $name    Dependency name.
+     * @param string $version Dependency version.
+     *
+     * @return void
      *
      * @codeCoverageIgnore - Slated for deprecation
      */
     public function setDependency($name, $version)
     {
-        $this->data['dependencies'][] = [
-            'name' => $name,
-            'version' => $version,
-        ];
+        $this->setEnvProperty($name, $version);
```

Comment:
Deprecated but make sure we're capturing the data here if it's being used anywhere else.